template/jiraDefaultTemplate.json

```diff
@@ -0,0 +1,132 @@
+{
+    "config": {
+        "jiraHostName": "boundary.jira.com",
+        "jiraPort": "",
+        "jiraUserName": "santosh",
+        "jiraPassword": "alliswell@100",
```

Comment:
Bad idea to put these credentials as default values in the template... even the host name has to be modified to something like `<hostname>`